selenium_tests/cross_listing/page_objects/cross_listing_index_page.py

```diff
@@ -0,0 +1,12 @@
+from selenium.webdriver.common.by import By
+
+from selenium_tests.cross_listing.page_objects.cross_listing_base_page_object\
+    import CrossListingBasePageObject
+
+
+class Locators(object):
+    HEADING_ELEMENT = (By.XPATH, '//h3[contains(.,"Cross Listing")]')
+
+
+class IndexPagePageObject(CrossListingBasePageObject):
```

Comment:
Nitpick -- do we need two `Page`s in the name? Also, is 'index page' sufficiently clear as to what's on the page? Maybe 'ListPageObject'? I dunno. Like I said, nitpick, not that important -- we can trace through the code to see what's what. Might be worth a comment though to explain what this PO is for (the `HEADING_ELEMENT` doesn't really give the reader an idea of what might be on the page -- although I'm sure as the PO is built out for test cases it will be more obvious).